model/response.py

```diff
@@ -143,6 +143,11 @@ def added_reactions(self):
              if not is_dummy(i['id'])]
         ))
 
+    def changed_reactions(self):
```

Comment:
I think everything that is currently under "changed" key should go to "measured" reactions. 
It raises additional questions though because it'll return all the measured reactions, including those that came from the "measurements" object from experimental data. But it looks like that's what would be beneficial for users, since it explicitly states what has been changed in the model, and even allows them to change it.  Maybe with that feature we will actually cover even more use cases! Let's put all the changed reaction to "measured" key and see how it'll look like on the frontend side.